### ISSUE TITLE
LDAP ingestion connector doesn't correctly assign groups 

### ISSUE THREAD

**Issue by jjoyce0510** [datahub-project/datahub#3335]
**Describe the bug**
When ingesting from LDAP connector, we are not using the GroupMembership aspect attached to the CorpUser. 

Instead, we are using the "members" list inside CorpGroupInfo, which has been deprecated. This ticket involves migrating the LDAP source to correctly populate the latest GroupMembership aspect.

**Comment by atulsaurav** [comment]
We use LDAP and this impacts us. @jjoyce0510 i can take this but would need a quick discussion.

**Comment by bda618** [comment]
I faced this issue few days ago while trying to ingest few users from a particular LDAP group (I used a filter to point out the the group). All CorpUser's were ingested correctly, but had no any information about GroupMembership. When I ingested a group, it did help. Eventually I solved this issue by creating a transformer which populated a GroupMembership aspect for CorpUsers.

The corp user could belong to 100+ groups, so I think it is vital to figured out which group you want to reference at GroupMembership. In my case it was a group at the filter, however, I am not sure about a general solution.
